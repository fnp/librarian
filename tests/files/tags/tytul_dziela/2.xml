<akap>
  Belzebub (z hebr.) --- w <tytul_dziela>Nowym Testamencie</tytul_dziela> pan piekieł i przywódca złych duchów. W utworze Krasińskiego ma on być twórcą ,,spróchniałego obrazu" raju, który to obraz --- niby dekorację teatralną --- złe duchy przechowują w sklepach, czyli podziemiach (staropolskie: <slowo_obce>sklep</slowo_obce> --- piwnica, loch, podziemie). Wokół interpretacji obrazu Edenu rozwinęła się polemika między A. Łuckim a Z. Niemojewską--Gruszczyńską (A. Łucki, <tytul_dziela>Obraz Edenu w <tytul_dziela typ="1">Nie--Boskiej komedii</tytul_dziela></tytul_dziela>, ,,Ruch literacki", nr 3, s.65--67; Z. Niemojewska--Gruszczyńska, <tytul_dziela>W sprawie obrazu Edenu w <tytul_dziela typ="1">Nie--Boskiej komedii</tytul_dziela></tytul_dziela>, tamże, nr 5, s. 133--135; A. Łucki, <tytul_dziela>Jeszcze w sprawie obrazu Edenu w <tytul_dziela typ="1">Nie--Boskiej komedii</tytul_dziela></tytul_dziela>, tamże, nr 6, s.191--192). Kleiner sądzi, że obraz Edenu to przyroda ,,zespolona z jakimiś utopijnymi marzeniami", ,,że plan pierwotny był odmienny", przyroda ,,miała być narzędziem szatana", kuszącego wyobraźnię poety. Za tą interpretacją idzie też wyd. Libery i Sawrymowicza. Według M. Janion interpretacja Edenu jako jednej z trzech wizji--kuszeń może być dwojaka. Po pierwsze można uznać, że Eden stanowi tu synonim natury, tak jak rozumieli ją romantycy: to raj utracony, stan naturalnej szczęśliwości, przeciwieństwo życia w cywilizacji, prosty i harmonijny żywot. Ale można rozumieć ów symboliczny Eden jako oświeceniowy ,,stan naturalny" człowieka związany z filozoficzną utopią XVIII--wiecznych racjonalistów. Wtedy pokusę szatańską łączyć należy z obrazem przyszłego szczęścia ludzkości, jaką roztacza Pankracy w rozmowie z hrabią Henrykiem.
</akap>
