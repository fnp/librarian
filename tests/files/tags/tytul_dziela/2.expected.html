<div id="book-text">
<a href="#f1" class="wl-num">1</a><p class="wl paragraph" id="f1">Belzebub (z hebr.) — w <em class="book-title">Nowym Testamencie</em> pan piekieł i przywódca złych duchów. W utworze Krasińskiego ma on być twórcą „spróchniałego obrazu” raju, który to obraz — niby dekorację teatralną — złe duchy przechowują w sklepach, czyli podziemiach (staropolskie: <em class="foreign-word">sklep</em> — piwnica, loch, podziemie). Wokół interpretacji obrazu Edenu rozwinęła się polemika między A. Łuckim a Z. Niemojewską–Gruszczyńską (A. Łucki, <em class="book-title">Obraz Edenu w <em class="book-title">„Nie–Boskiej komedii”</em></em>, „Ruch literacki”, nr 3, s.65–67; Z. Niemojewska–Gruszczyńska, <em class="book-title">W sprawie obrazu Edenu w <em class="book-title">„Nie–Boskiej komedii”</em></em>, tamże, nr 5, s. 133–135; A. Łucki, <em class="book-title">Jeszcze w sprawie obrazu Edenu w <em class="book-title">„Nie–Boskiej komedii”</em></em>, tamże, nr 6, s.191–192). Kleiner sądzi, że obraz Edenu to przyroda „zespolona z jakimiś utopijnymi marzeniami”, „że plan pierwotny był odmienny”, przyroda „miała być narzędziem szatana”, kuszącego wyobraźnię poety. Za tą interpretacją idzie też wyd. Libery i Sawrymowicza. Według M. Janion interpretacja Edenu jako jednej z trzech wizji–kuszeń może być dwojaka. Po pierwsze można uznać, że Eden stanowi tu synonim natury, tak jak rozumieli ją romantycy: to raj utracony, stan naturalnej szczęśliwości, przeciwieństwo życia w cywilizacji, prosty i harmonijny żywot. Ale można rozumieć ów symboliczny Eden jako oświeceniowy „stan naturalny” człowieka związany z filozoficzną utopią XVIII–wiecznych racjonalistów. Wtedy pokusę szatańską łączyć należy z obrazem przyszłego szczęścia ludzkości, jaką roztacza Pankracy w rozmowie z hrabią Henrykiem.</p>
</div>
